seleniumbase/core/browser_launcher.py

```diff
@@ -807,6 +815,7 @@ def get_driver(
             locale_code,
             servername,
             proxy_string,
+            proxy_bypass_list,
             proxy_auth,
             proxy_user,
             proxy_pass,
```

Comment:
Looks like ``proxy_bypass_list`` is not in the correct location here. Should be after ``proxy_pass``.